hstrat/serialization/_pack_differentiae.py

```diff
@@ -0,0 +1,39 @@
+from base64 import b64encode
+import typing
+
+from bitstring import BitArray
+
+from ..genome_instrumentation import HereditaryStratum
+
+
+def pack_differentiae(
+    strata: typing.Iterable[HereditaryStratum],
+    differentia_bit_width: int,
+) -> str:
+
+    # this is a concatenation, not a mathematical sum
+    differentia_bits = sum(
+        (
+            BitArray(
+                uint=stratum.GetDifferentia(),
+                length=differentia_bit_width,
+            )
+            for stratum in strata
+        ),
+        BitArray(),
+    )
+
+    # use first byte to tell how many null bits if any are possible
```

Comment:
possible -> required

null -> padding (also in variable names below)

Reply:
Done in 31ac9c6.